test/fennec/udp_test.exs

```diff
@@ -228,8 +230,12 @@ defmodule Fennec.UDPTest do
   end
 
   defp udp_communicate(udp, client_id, req) do
+    with_mock Fennec.Auth, [:passthrough], [
```

Comment:
I see. Let's leave it for now. I hope we'll get to remove it int he future (when we have something like pluggable auth modules).